Dockerfile

```diff
@@ -46,7 +46,7 @@ COPY --from=builder /app/res/ /res/
 COPY --from=builder /app/static/ /static/
 COPY --from=builder /app/app-rfid-llrp-inventory /app-rfid-llrp-inventory
 
-EXPOSE 48086
+EXPOSE 59711
```

Comment:
Add the following just before this line to enable write access for the volume.

```
RUN mkdir -p /cache && \
    chown -R 2002:2002 /cache

VOLUME /cache
```

Reply:
I have added those lines to the docker file